app/src/ui/multi-commit-operation/reorder.tsx

```diff
@@ -3,6 +3,7 @@ import { BaseRebase } from './base-rebase'
 
 export abstract class Reorder extends BaseRebase {
   protected conflictDialogOperationPrefix = 'reordering commits on'
+  protected rebaseKind = MultiCommitOperationKind.Reorder
```

Comment:
The reorder and squash "worked" without this, but in the conflict dialog after a rebase has been continued ones (2nd time hitting rebase dialog) you wouldn't see the "Reorder Commit" or "Squash Commit".